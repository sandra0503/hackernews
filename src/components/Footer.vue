<template>
  <footer class="footer">2019 Hackernews</footer>
</template>

<script>
export default {
  name: "Footer"
};
</script>

<style lang="scss" scoped="true">
.footer {
  background: #009688;
  color: #fff;
  font-size: 14px;
  padding: 20px;
}
</style>
