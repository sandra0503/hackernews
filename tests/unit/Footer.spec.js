import { shallowMount } from "@vue/test-utils";
import Footer from "@/components/Footer";

describe("Footer component", () => {
  const wrapper = shallowMount(Footer);

  it("renders the component and matches snapshot", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
