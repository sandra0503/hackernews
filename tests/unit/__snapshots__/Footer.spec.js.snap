// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders the component and matches snapshot 1`] = `<footer class="footer">2019 Hackernews</footer>`;
